### PR TITLE
chore: order of releases in changelog

### PR DESCRIPTION
**Pull request by 2color**

### CODE REVIEW

**Comment by achingbrain** [comment]
We've been through a couple of different release tools which format things differently, and code has been moved around without trimming the release logs, it's a bit of a mess.

File: packages/helia/CHANGELOG.md [hunk 1]
```diff
@@ -187,6 +173,19 @@
...
+### Dependencies
+
+* **dev:** bump aegir from 39.0.13 to 40.0.8 ([#198](https://github.com/ipfs/helia/issues/198)) ([4d75ecf](https://github.com/ipfs/helia/commit/4d75ecffb79e5177da35d3106e42dac7bc63153a))
+* update sibling dependencies ([beb10b5](https://github.com/ipfs/helia/commit/beb10b5590d66d1d5bef9b5e890b888263df2c92))
+
+### Dependencies
+
+* The following workspace dependencies were updated
+  * dependencies
+    * @helia/block-brokers bumped from ^2.0.0 to ^2.0.1
```
Comment (SgtPooki) [inline packages/helia/CHANGELOG.md:188]: it looks like there's a trailing `dependencies` below this that's actually a part of `### [2.0.1](https://www.github.com/ipfs/helia/compare/helia-v2.0.0...helia-v2.0.1) (2023-08-16)`

**Review by SgtPooki — CHANGES_REQUESTED** [review]
we should probably try to re-generate the changelog completely using semantic-release.

**Comment by SgtPooki** [comment]
@achingbrain if we remove the changelogs in https://github.com/ipfs/helia/pull/483 can we get release-please to regenerate the full changelog?

**Comment by achingbrain** [comment]
I don't know tbh, but my guess would be no.  I think it only considers changes since the last release tag, otherwise we'd see CI releases failing all over the place due to GitHub applying rate limiting (this happens when you stuff up the release-please config, for example).

**Comment by achingbrain** [comment]
Ok, I've updated the formatting of all changelogs to be consistent with the latest release tool.

I've also re-tagged all the old releases from the `@helia/foo-vNNN` format to `foo-vNNN` and removed the old tags so everything's consistent.

Please delete your local tags in any cloned copy of helia you have and fetch the updated list (this will prevent you re-creating the old tag formats if you accidentally run `git push --tags`, for example):

```console
$ git tag -l | xargs git tag -d
$ git fetch
```

Hopefully going forwards everything autogenerated will be done just so.

**Review by achingbrain — APPROVED** [review]

**Pull request merged**